ui/i18n/src/main/res/values/strings.xml

```diff
@@ -286,20 +286,20 @@
         <item quantity="other">%d downloads left</item>
     </plurals>
     <string name="completing">Completing…</string>
-    <string name="download_notification_title">Downloading podcast data</string>
-    <string name="download_notification_title_feeds">Refreshing podcasts</string>
-    <string name="download_notification_title_episodes">Downloading episodes</string>
+    <string name="download_notification_title">Downloading Podcast Data</string>
+    <string name="download_notification_title_feeds">Refreshing Podcasts</string>
+    <string name="download_notification_title_episodes">Downloading Episodes</string>
     <string name="download_log_title_unknown">Unknown Title</string>
     <string name="download_type_feed">Feed</string>
     <string name="download_type_media">Media file</string>
     <string name="null_value_podcast_error">No podcast was provided that could be shown.</string>
     <string name="no_feed_url_podcast_found_by_search">The suggested podcast did not have an RSS link, AntennaPod found a podcast that could match</string>
-    <string name="authentication_notification_title">Authentication required</string>
+    <string name="authentication_notification_title">Authentication Required</string>
     <string name="authentication_notification_msg">The resource you requested requires a username and a password</string>
     <string name="confirm_mobile_download_dialog_title">Confirm Mobile Download</string>
```

Comment:
For dialog titles I would say we follow the Google policy. Maybe we should discuss again on the forum.